frontend/src/components/Participants.vue

```diff
@@ -0,0 +1,288 @@
+        v-model="warningAlert"
+        dismissible
+        @click="warningAlert = !warningAlert"
+        transition="slide-y-transition"
+        >{{ warningMessage }}</v-alert
+      >
+      <v-alert
+        type="success"
+        :value="successAlert"
+        v-model="successAlert"
+        dismissible
+        @click="successAlert = !successAlert"
+        transition="slide-y-transition"
+        >Participant has been successfully added</v-alert
+      >
+    </v-flex>
+    <v-card>
+      <v-toolbar flat>
+        <v-icon class="pa-3">fas fa-user-plus</v-icon>
+        <v-toolbar-title>Create new participant</v-toolbar-title>
+      </v-toolbar>
+      <v-divider></v-divider>
+      <v-form @keyup.enter="onSubmit" class="pa-6" ref="form">
+        <v-text-field
+          type="text"
+          label="E-mail"
+          v-model.trim="form.email"
+          @blur="$v.form.email.$touch()"
+          :error-messages="emailErrors"
+        />
+        <v-text-field
+          label="Github link"
+          v-model="form.github"
+          @blur="$v.form.github.$touch()"
+          :error-messages="githubErrors"
+        />
+        <v-text-field
+          label="First name"
+          v-model="form.first_name"
+          @blur="$v.form.first_name.$touch()"
+          :error-messages="firstNameErrors"
+        />
+        <v-text-field
+          label="Last name"
+          v-model="form.last_name"
+          @blur="validateLastName($event)"
+          :error-messages="lastNameErrors"
+        />
+        <v-text-field
+          label="Slack nick"
+          v-model="form.slack"
+          @blur="validateSlack($event)"
+          :error-messages="slackErrors"
+        />
+        <v-text-field
+          label="Phone No."
+          v-model="form.phone"
+          :counter="9"
+          @blur="validatePhone($event)"
+          :error-messages="phoneErrors"
+        />
+        <v-card-actions class="pt-3 align-center justify-center">
+          <v-btn
+            :disabled="$v.$invalid"
+            @click="onSubmit"
+            class="add-participant-btn align-center justify-center"
+            >Add new participant</v-btn
+          >
+        </v-card-actions>
+      </v-form>
+    </v-card>
+  </v-container>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import { mapGetters } from 'vuex';
+import {
+  required,
+  minLength,
+  maxLength,
+  email,
+  numeric
+} from 'vuelidate/lib/validators';
+export default {
+  data() {
+    return {
+      form: {
+        email: '',
+        github: '',
+        first_name: '',
+        last_name: '',
+        slack: '',
+        phone: ''
+      },
+      warningAlert: false,
+      warningMessage: '',
+      successAlert: false
+    };
+  },
+  created() {
+    this.$store.dispatch('participant/getParticipants');
+    setInterval(() => {
+      this.successAlert = false;
+    }, 10000);
+    setInterval(() => {
+      this.warningAlert = false;
+    }, 30000);
+  },
```

Comment:
Maybe instead of set intervals running all the time in the background, we could implement that differently ? For example these same behaviour could be achieved using setTimeout function inside methods.

Reply:
Seems reasonable. I'll remember reading about it but for some reason I gave up on this. Don't remember why. I'll try again

Reply:
Ok it worked - changes committed.